src/openapi/peridio-admin-openapi.yaml

```diff
@@ -2914,14 +2905,11 @@ components:
           $ref: "#/components/schemas/role-name"
         username:
           $ref: "#/components/schemas/user-username"
-    page:
-      description: |
-        A cursor for pagination across multiple pages of results. Don’t include this parameter on
-        the first call. Use the `next_page` value returned in a previous response to request
-        subsequent results.
-      type: string
     platform:
       type: string
+    prn:
+      description: TODO
```

Comment:
@danielspofford I merged all prns into a global prn, but I'm unsure what to document here